fs_storage/models/fs_storage.py

```diff
@@ -408,7 +408,8 @@ def find_files(self, pattern, relative_path="", **kw) -> list[str]:
             return []
         regex = re.compile(pattern)
         for file_path in self.fs.ls(relative_path, detail=False):
-            if regex.match(file_path):
+            # fs.ls returns a relative path
```

Comment:
As this method is deprecated, does it makes sense to improve it? Unless, this is a fix...

Reply:
yes, it's a fix